tutorials/Example_Track_on_Radar_Segment_on_Satellite/track_on_radar_segment_on_satellite.ipynb

```diff
@@ -0,0 +1,1245 @@
+      "text/plain": [
+       "<Figure size 800x800 with 3 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# In Py-ART's graphing suite, there is a display class similar to RadarMapDisplay,\n",
+    "# but for grids. To plot the grid:\n",
+    "fig = plt.figure(figsize=[8, 8])\n",
+    "plt.axis('off')\n",
+    "\n",
+    "display = pyart.graph.GridMapDisplay(grid)\n",
+    "display.plot_grid('reflectivity', level=3, vmin=0, vmax=60, fig=fig)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "xr_grid = grid.to_xarray()"
```

Comment:
This variable naming is a bit obscure, could be renamed to something like `radar_grid` to calrify what it is